spec/Coduo/PHPHumanizer/Collection/FormatterSpec.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace spec\Coduo\PHPHumanizer\Collection;
+;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\Translation\Translator;
+
+class FormatterSpec extends ObjectBehavior
+{
+    function let(Translator $translator)
+    {
+        $this->beConstructedWith($translator);
+        $translator->trans(
+            'only_two',
+            array('%first%' => 'Michal', '%second%' => 'Norbert'),
+            'oxford'
+        )->willReturn('Michal and Norbert');
+
+        $translator->transChoice(
+            'comma_separated_with_limit',
+            1,
+            array('%count%' => 1, "%list%" => 'Michal, Norbert'),
+            'oxford'
+        )->willReturn('Michal, Norbert and 1 more');
+
+        $translator->trans(
+            'comma_separated',
+            array("%list%" => 'Michal, Norbert', '%last%' => "Lukasz"),
+            'oxford'
+        )->willReturn('Michal, Norbert and Lukasz');
```

Comment:
Ur right, but some ppl in different communites always asked for ability to specify if you need comma or not, so the format is defined in translation file. In that case everyone can adjust this to their own locale. Tho maybe I should add the default comma in translation file.